src/pipeline.ts

```diff
@@ -320,9 +331,17 @@ export class GitHubWorkflow extends PipelineBase {
       });
     };
 
+    const replaceAssetHash = (template: string) => {
+      const hash = path.parse(template.split('/').pop() ?? '').name;
```

Comment:
this is fairly brittle; is it ok? are there alternatives that I can explore here?

Reply:
I bet there are cases where it's going to fail, but we'll deal with those when we come to them.